Backpack/HorizontalNavigation/Classes/BPKHorizontalNavigation.m

```diff
@@ -191,8 +198,13 @@ - (void)setSize:(BPKHorizontalNavigationSize)size {
 
 #pragma mark - Private
 
+// This should be replaced with BpkBorderSizeXl when it's available
 - (CGFloat)barHeight {
-    return BPKSpacingSm / 2;
+    return 3;
+}
+
+- (CGFloat)barSpacing {
+    return 3 * BPKSpacingSm;;
```

Comment:
```suggestion
    return 3 * BPKSpacingSm;
```

Reply:
I can catch this in my follow up PR (https://github.com/Skyscanner/backpack-ios/pull/540) if you prefer?